### DISCUSSION TITLE
Chai Stack ☕ (Nativewind, expo-router, zustand, typescript)

### DISCUSSION THREAD

**Discussion by AviAvinav** [nativewind/nativewind#452] [Show and tell]
I made a stack with nativewind, expo-router, zustand and typescript.

Repo: https://github.com/aviAvinav/chai

**Comment by truejordan** [comment]
HI greta job. Im having issues with using nativewind and expo-router in my stack. I get the error cant find module 'nativewind/metro'. Did you face this issue and how did you fix this?

**Reply by AviAvinav** [reply]
No, I didn't have any such issue, what version of nativewind are you using? You can also try `npx expo start -c`, maybe it will work. I can probably help you more if you share the error log.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
